### PR TITLE
added insert step

### PR DESCRIPTION
**Pull request by eacharles**

### CODE REVIEW

File: src/lsst/cm/tools/cli/commands.py [hunk 1]
```diff
@@ -65,6 +65,34 @@ def insert(
...
+@options.campaign()
+@options.config_block()
+@options.script_method()
+def insert_step(
+    dbi: DbInterface,
+    config_block: str,
+    script_method: ScriptMethod,
+    **kwargs: Any,
+) -> None:
+    """Insert a new database entry at a particular level"""
```
Comment (eigerx) [inline src/lsst/cm/tools/cli/commands.py:81]: Probably want it to say "at the step level"

File: src/lsst/cm/tools/db/sqlalch_interface.py [hunk 2]
```diff
@@ -209,6 +210,43 @@ def insert(
...
 
+    def insert_step(
+        self,
+        parent_db_id: DbId,
+        config_block: str,
+        **kwargs: Any,
+    ) -> CMTableBase:
+        parent_level = parent_db_id.level()
+        campaign = self.get_entry(parent_level, parent_db_id)
+        config = campaign.config_
```
Comment (eigerx) [inline src/lsst/cm/tools/db/sqlalch_interface.py:221]: I see where we get the fix for the input collection. Is this part the fix for setting the LSST_VERSION etc?

**Review by eigerx — APPROVED** [review]
Is there a line here somewhere that also makes sure the pipeline_yaml gets into the bps yaml? Or was that an error on our part?

File: src/lsst/cm/tools/db/sqlalch_interface.py (hunk 2 above)
Comment (eacharles) [reply src/lsst/cm/tools/db/sqlalch_interface.py:221]: it should get pulled from parent object.

**Pull request merged**